src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -439,6 +439,17 @@ public static bool IsTargetReadyForAeWithKeyStore()
                 ;
         }
 
+        public static bool IsSupportingDistributedTransactions()
```

Comment:
I tried to use the same `Is` pattern although I felt is is a bit of a clumsy name. I would have preferred `SupportsDistributedTransactions` but I wanted to make sure it fits the style of other such methods.